.github/workflows/CI.yml

```diff
@@ -3,11 +3,11 @@ on:
   push:
     branches:
       - "*"
```

Comment:
question: Does '*' does not take care of all the branch names?

Reply:
Unfortunately no. it will not match "/".